.azure-pipelines/win32-conda-CI.yml

```diff
@@ -1,113 +1,113 @@
-# Python package
```

Comment:
What's the change in the file? all lines look updated

Reply:
Windows probably changed the end of line. I need to restore it.

Reply:
Fixed.